src/database/migrations/20210106142657-create-people-table.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("people", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
```

Comment:
- I would make it here UNSIGNED BIGINT. ID's in a table will never be negative values (https://stackoverflow.com/questions/3895692/what-does-unsigned-in-mysql-mean-and-when-to-use-it).